### PR TITLE
Run windows agent as LOCAL_SERVICE

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?
Use the built in account "LOCAL_SERVICE" instead of self-made account "ddagentuser".

### Motivation
Self made account creation has proven less reliable that we'd like

### CODE REVIEW

File: omnibus/resources/agent/msi/cal/CustomAction.cpp [hunk 1]
```diff
@@ -40,162 +32,16 @@ extern "C" UINT __stdcall FinalizeInstall(MSIHANDLE hInstall) {
...
-    //   If user is NOT present
-    //     if service is present
-    //       ERROR how could service be present but user not present?
-    //     if service is not present
-    //       new install in this domain
-    //       must have password for user creation and service installation
-    //
-    // If NOT a domain controller
-    //   if user is present
-    //     if the service is present
```
Comment (ogaca-dd) [inline omnibus/resources/agent/msi/cal/CustomAction.cpp]: Note: The following functions are not called anymore:
 - generatePassword
 - doesUserExist
 - setUserProfileFolder
 - getUserProfileFolder
 - AddPrivileges

File: omnibus/resources/agent/msi/cal/cleanupgrade.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+     
+    WcaLog(LOGMSG_STANDARD, "Initialized.");
+    initializeStringsFromStringTable();
+    DoStopSvc(hInstall, agentService);
+    er = doDDUninstallAs(hInstall, UNINSTALL_UNINSTALL);
+    if (er != 0) {
+        hr = -1;
+    }
+LExit:
+    er = SUCCEEDED(hr) ? ERROR_SUCCESS : ERROR_INSTALL_FAILURE;
```
Comment (ogaca-dd) [inline omnibus/resources/agent/msi/cal/cleanupgrade.cpp]: Please note you did not remove anymore the home folder for `ddagentuser` (only an issue when upgrading from 6.14 to 6.15).
If it is not an issue you can also remove the functions: RemoveUserProfile, RemoveRegisterKey, RemoveLockedFolder and getLastErrorMessage.

**Review by ruthnaebeck — COMMENTED** [review]
Doesn't appear to need docs review

**Comment by djmitche** [comment]
@derekwbrown this looks like it has a lot of conflicts!  Is this something that should be tracked to completion elsewhere, or should we close the PR?

**Pull request closed without merging**